test/over_react/util/react_wrappers_test.dart

```diff
@@ -42,27 +42,29 @@ main() {
         'originalPropToOverride': 'original'
       };
 
+      JsBackedMap testPropsForComponent2 = JsBackedMap.from(testProps);
```

Comment:
We don't want to require consumers to pass in a `JsBackedMap` when cloning a Component2, so can we remove this and use `testProps` instead?